### PR TITLE
#28 Feature:  Implementing landing-page.

### PR DESCRIPTION
**Pull request by Niyonkuru1**
What does this PR do?

- Create UI for landing-page

Description of Task to be completed?

- Landing Page UI is created. It includes three components; footer, navbar and LandingPage.  

- It should be responsive.

- It should perfectly resemble to the corresponding design in the mockup

-  UI skeleton for loading displayed

How should this be manually tested?
- Clone the repo
- checkout to branch ft-landing-page-28
- pull the changes by git pull
- install all packages by running npm install
- start app by running npm run start.
-  It should dispaly all contents of the landing page as depicted in the design

Any background context you want to provide?

- [ Work In Progress
 ]; 

What are the relevant pivotal tracker stories?
[Trello Board # 28](https://trello.com/c/7MYeedeR/28-landing-page)

Screenshots (if appropriate)

### CODE REVIEW

**Review by tysepa — APPROVED** [review]

**Review by PrinceNiyonshuti — APPROVED** [review]
Looks good

File: packageDraft.txt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+//     "dotenv-webpack": "^7.1.0",
+//     "jest": "^27.5.1",
+//     "path": "^0.12.7",
+//     "react": "^18.0.0",
+//     "react-dom": "^18.0.0",
+//     "react-redux": "^7.2.8",
+//     "react-router": "^6.3.0",
+//     "react-router-dom": "^6.3.0",
+//     "react-scripts": "^5.0.0",
+//     "react-test-renderer": "^18.0.0",
```
Comment (thepatrickniyo) [inline packageDraft.txt]: Can you please try to remove the comments.

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]
Remove the commented source codes then we be able to merge the work

**Review by Hodal66 — APPROVED** [review]

**Review by PrinceNiyonshuti — APPROVED** [review]
looks good

**Comment by thepatrickniyo** [comment]
Remove the conflicts in this branch

**Review by thepatrickniyo — APPROVED** [review]

**Pull request merged**